examples/nlp/text_classification/config.yaml

```diff
@@ -0,0 +1,58 @@
+# Sentence classification with pretrained BERT models
+batch_size: 32
+
+# IO
+work_dir: ./outputs
+checkpoint_dir: null
+
+# Data
+data_dir: /raid/data/nlp/SST-2 #/path/to/data
+train_file_prefix: train
+eval_file_prefix: dev
+shuffle: true # shuffle training data
+num_train_samples: -1
+num_val_samples: -1
+num_workers: 2
+pin_memory: false
+use_cache: true
+
+# Language Model
+pretrained_model_name: roberta-base
+bert_checkpoint: null
+bert_config: null
+# TODO: we can specialize config based on other config values at runtime:
+# https://hydra.cc/docs/next/patterns/specializing_config
+tokenizer: nemobert # only used if using custom checkpoint
+tokenizer_model: null # only used if tokenizer is sentencepiece
+do_lower_case: null # true for uncased models, false for cased models
```

Comment:
Shouldn't it be has default value of False?

Reply:
For text classification don't we normally have do_lower_case=True?

Reply:
We can set the default to True, but my question is about the null value. What exactly does it mean? False or None?

Reply:
null means None